anubis-management-api/src/main.py

```diff
@@ -1,6 +1,7 @@
 from fastapi import Depends, FastAPI
 from tenants import routers as t
 from policies import routers as p
+from fastapi.middleware.cors import CORSMiddleware
```

Comment:
as said above, following gabriele's pr, code in the master has already cors support.
so remove.